docs/react-navigation.md

```diff
@@ -8,10 +8,10 @@ React-navigation does not unmount components when switching between tabs. So whe
 
 ```jsx
 
-import { withNavigationFocus } from 'react-navigation' 
+import { useIsFocused } from '@react-navigation/core';
 
  render() {
-    const { isFocused } = this.props
+    const isFocused = useIsFocused();
```

Comment:
Hooks cannot be used in a class component. Making the example a function component would be ideal.

Reply:
Ah yes I missed that. Will do.

Reply:
I think I have changed it properly now.